examples/ftd_api_scripts/timerange.py

```diff
@@ -0,0 +1,110 @@
+    """
+    End to end example of code that updates a time-range object.
+    Requires Python v3.0 or greater and the reqeusts library.
+    You must update the values for host, port, user, and password to connect to your device.
+    """
+
+    access_token = get_access_token(host, port, user, passwd)
+    if not access_token:
+        print("Unable to obtain an access token. Did you remember to set host, port, user, and password?")
+        return
+    time_range = get_all_time_range(host, port, access_token)
+    if not time_range:
+        print('Unable to get time range')
+    tro = {
+    "name": "TRO_Python_Request_Test_7",
+    "description": "creating Time Range",
+    "effectiveStartDateTime": "2020-07-24T12:08",
+    "effectiveEndDateTime": "2020-07-28T12:08",
+    "recurrenceList": [
+    {
+      "recurrenceType": "DAILY_INTERVAL",
+      "days": ["MON","TUE","FRI"],
+      "dailyStartTime": "08:00",
+      "dailyEndTime": "09:00",
+      "type" : "recurrence"
+    },
+    {
+      "recurrenceType": "RANGE",
+      "rangeStartTime": "09:00",
+      "rangeEndTime": "11:00",
+      "rangeStartDay": "MON",
+      "rangeEndDay": "TUE",
+      "type" : "recurrence"
+    }
+  ],
+  "type": "timerangeobject"
+  }
+    time_range = post_time_range(host, port, access_token, tro)
+    if not time_range:
+        print('Unable to create time range object')
+        return
+    tro_update = {
+    'version': time_range['version'],
+    "name": "TRO_Python_Request_Test_Name_updated_7",
+    "description": "editing Time Range",
+    "effectiveStartDateTime": "2019-11-24T12:08",
+    "effectiveEndDateTime": "2019-12-28T12:08",
+    "recurrenceList": [
+    {
+      "recurrenceType": "DAILY_INTERVAL",
+      "days": ["MON","TUE","FRI"],
+      "dailyStartTime": "08:00",
+      "dailyEndTime": "09:00",
+      "type" : "recurrence"
+    },
+    {
+      "recurrenceType": "RANGE",
+      "rangeStartTime": "09:00",
+      "rangeEndTime": "11:00",
+      "rangeStartDay": "MON",
+      "rangeEndDay": "TUE",
+      "type" : "recurrence"
+    }
+  ],
+  "type": "timerangeobject"
+    }
+    updated_tro = update_time_range(host, port, access_token, time_range['id'], tro_update)
+    if not updated_tro:
+        print('Unable to update time range object')
+        return
+    tro = get_time_range(host, port, access_token, updated_tro['id'])
+    if not tro:
+        print('Unable to get time range object')
+        return
+    print('TRO name is {}'.format(tro['name']))
+    returnCode = delete_time_range(host, port, access_token, tro['id'])
+    if not returnCode:
+        print('Unable to delete time range object')
+        return
+
+if __name__ == '__main__':
+
+    if len(sys.argv) != 5:
+        print("Usage: python cookbook_scripts/timerange.py host port user passwd")
+        exit(1)
+
+    host = sys.argv[1]
+    port = sys.argv[2]
+    user = sys.argv[3]
+    passwd = sys.argv[4]
+    time_range_actions(host, port, user, passwd)
```

Comment:
Please add exit(0) and exit(1) depending on return value from time_range_actions()

Reply:
Done, addressed in https://github.com/stleary-cisco/ftd-rest-api-cookbook/pull/7